pages/database-management/monitoring.mdx

```diff
@@ -7,50 +7,102 @@ import { Callout } from 'nextra/components'
 
 # Monitoring
 
-In production systems, monitoring of applications is crucial, and that includes
-databases as well. Memgraph allows monitoring the database via WebSocket server and a
-HTTP server. 
+Monitoring of applications and databases is essential to be able to track the
+load, resources, logs, transactions, latency etc. 
```

Comment:
Avoid using "etc". Give the correct, exhaustive list of things the user must consider.